ts/components/game/Card.tsx

```diff
@@ -58,6 +59,7 @@ class Card extends React.Component<CardProps, {}> {
 		var defaultHealth = null;
 		var defaultDurability = null;
 		var cardType = entity.getCardType();
+		var cardClass = entity.getClass();
```

Comment:
I may be misunderstanding how this should work, but it seems to be always 0. Used the card data below instead.